docs/content/server/setup.md

```diff
@@ -19,121 +19,99 @@ We also offer this in video format. You can check it out [here](https://github.c
 
-In order to setup the database, you need to run the following commands, in the same terminal you ran the `poetry install` command:
+* Copy the `.env.example` file available in the `supabase/docker` directory to `.env` in `autogpt_platform`:
+  ```
+   cp supabase/docker/.env.example .env
+  ```
+  This command will copy the `.env.example` file to `.env` in the `supabase/docker` directory. You can modify the `.env` file to add your own environment variables.
 
-   ```sh
-   docker compose up postgres redis -d
-   poetry run prisma migrate dev
-   ```
-After deploying the migration, to ensure that the database schema is correctly mapped to your codebase, allowing the application to interact with the database properly, you need to generate the Prisma database model:
+* Run the backend services:
+  ```
+   docker compose up -d
+  ```
+  This command will start all the necessary backend services defined in the `docker-compose.combined.yml` file in detached mode.
 
-```bash
-poetry run prisma generate
-```
 
-Without running this command, the necessary Python modules (prisma.models) won't be available, leading to a `ModuleNotFoundError`.
+## Running the frontend application
 
-### Get access to Supabase
+To run the frontend application, follow these steps:
 
-Navigate to rnd/supabase
-Run the following command:
+* Navigate to `frontend` folder within the `autogpt_platform` directory:
+  ```
+   cd frontend
+  ```
 
-```bash
-  git submodule update --init --recursive
-```
-### Running the server 
+* Copy the `.env.example` file available in the `frontend` directory to `.env` in the same directory:
+  ```
+   cp .env.example .env
+  ```
+  You can modify the `.env` within this folder to add your own environment variables for the frontend application.
 
-To run the server, navigate back to rnd (cd..) and run the following commands in the same terminal you ran the `poetry install` command:
-
-```bash
-cp supabase/docker/.env.example .env
-docker compose build
-docker compose up -d
-```
-
-In the other terminal from frontend, you can run the following command to start the frontend:
-
-```bash
-cp .env.example .env
-yarn dev
-```
+* Run the following command:
+  ```
+   npm install
+   npm run dev
+  ```
+  This command will install the necessary dependencies and start the frontend application in development mode.
 
-### Checking if the server is running
+## Checking if the application is running
 
 You can check if the server is running by visiting [http://localhost:3000](http://localhost:3000) in your browser.
 
 ### Notes: 
-By default the daemons for different services run on the following ports: 
+By default the appplication for different services run on the following ports: 
 
-Execution Manager Daemon: 8002
-Execution Scheduler Daemon: 8003
-Rest Server Daemon: 8004
+Frontend UI Server: 3000
+Backend Websocket Server: 8001
+Backend API Rest Server: 8006
```

Comment:
Marketplace API Server: 8015

Also

Prisma studio can be accessed from `http://127.0.0.1:8000` Using the default login credentials:

```
DASHBOARD_USERNAME=supabase
DASHBOARD_PASSWORD=this_password_is_insecure_and_should_be_updated
```

Reply:
Marketplace API should be eventually replaced with the cloud version and it doesn't make sense to use the the local version of it.

Also the one running on `http://127.0.0.1:8000` is supabase studio not prisma, and currently it's also not working.